src/main/java/com/softserve/sprintone/ivan/TaskIvanSprintOne.java

```diff
@@ -0,0 +1,422 @@
+package com.softserve.sprintone.ivan;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.Math.pow;
+import static java.lang.Math.random;
+
+/**
+ * @author Iván Caballero
+ */
+
+/**
+ * Task 13 - The code provided has a method plus100 which is supposed to return the number of the
+ * parameter incremented by 100. But it's not working properly.
+ * Constraints:
+ * - Do not modify the constructor
+ * - Do not modify the plus100 method
+ * - Do not modify the ONE_HUNDRED declaration
+ */
+class Dinglemouse {
+    private static final int ONE_HUNDRED = 100;
+    public static final Dinglemouse dinglemouse = new Dinglemouse();
```

Comment:
if it is a constant field, then it should follow Java conventions:

"The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_"). (ANSI constants should be avoided, for ease of debugging.)"

DINGLE_MOUSE or DINGLEMOUSE if a single word.

By the way, what does it mean "dingle mouse"?

Reply:
It was a code provided in the codewars exercises and the task was about fix a tiny problem in the declaration of that class so leave the code as it was in the page. Just copy paste